persistence/ravendb/outbox_disable-cleanup_raven_[4,).partial.md

```diff
@@ -0,0 +1,3 @@
+The cleanup task can be disabled by specifying a large value for `SetFrequencyToRunDeduplicationDataCleanup`, for example `TimeSpan.FromYears(20). This can be useful when an endpoint is scaled out and instances are competing to run the cleanup task.
+
```

Comment:
Same thing here: The current behavior is that cleanup still runs once after one 1 minute after the endpoint instance started no matter the configure frequency.

Reply:
This is covered in the note below but I think it's slightly different. In this version, the cleanup is executed immediately. I didn't see anything in the code to indicate it will be delayed.